public/_data/repositories/core.yml

```diff
@@ -0,0 +1,39 @@
+section:
+  description: These are the primary repositories for Conjur Open Source and its SDK.
+  categories:
+  - name: Conjur OSS Core
```

Comment:
Category headers like this are not links, yet they have the same color/styling as repo links.  Any chance of links could have a unique style?

Reply:
I think that's a great idea - I might start a doc to track the changes that we want for now. This change I probably won't make until we migrate to wordpress (as planned at some future date)